x-pack/plugins/apm/public/components/app/settings/agent_configurations/agent_configuration_create_edit/service_page/service_page.tsx

```diff
@@ -162,7 +201,8 @@ export function ServicePage({ newConfig, setNewConfig, onClickNext }: Props) {
             isDisabled={
               !newConfig.service.name ||
               !newConfig.service.environment ||
-              agentNameStatus === FETCH_STATUS.LOADING
+              agentNameStatus === FETCH_STATUS.LOADING ||
+              !isAgentConfigurationSupported
```

Comment:
[nit] this is growing, could we maybe move it to a const?

Reply:
Fixed